### PR TITLE
[DOC-2981] HCE versus Litmuschaos

### PR DESCRIPTION
**Pull request by SmritiSatya**
[**Link to Preview**](https://646f2e1eadf0ce2a55b181bb--harness-developer.netlify.app/docs/chaos-engineering/get-started/powered-by-litmus)

### CODE REVIEW

File: docs/chaos-engineering/technical-reference/hce-versus-litmus.md [hunk 1]
```diff
@@ -0,0 +1,829 @@
...
+        <td>✅</td>
+        <td>✅</td>
+    </tr>
+    <tr>
+        <td>Run chaos faults in parallel</td>
+        <td>✅</td>
+        <td>✅</td>
+    </tr>
+    <tr>
+        <td>Experiment pontrol arameters through the tag in the UI</td>
```
Comment (StrawberryPad) [inline docs/chaos-engineering/technical-reference/hce-versus-litmus.md:40]:
```suggestion
        <td>Experiment control parameters through the tag in the UI</td>
```

File: docs/chaos-engineering/technical-reference/hce-versus-litmus.md [hunk 2]
```diff
@@ -0,0 +1,829 @@
+---
+title: Harness Chaos Engineering (HCE) versus LitmusChaos
+sidebar_position: 2
+---
+This section describes the differences between Harness Chaos Engineering (HCE) and the open-source CNCF incubation project, LitmusChaos.
+
+### Chaos management- Basic 
+<table>
```
Comment (StrawberryPad) [inline docs/chaos-engineering/technical-reference/hce-versus-litmus.md:8]: For your next iteration of these tables, from a Harness perspective it's better to switch the columns so HCE is to the left of litmus (reverse these two columns), and so the one that has more features/green checkmarks is on the left of the one that has less features/more red Xs. :-)

**Review by StrawberryPad — DISMISSED** [review]

**Comment by ksatchit** [comment]
Would be great to get the review from @umamukkara  on this.

**Comment by StrawberryPad** [comment]
@umamukkara - please review

**Review by umamukkara — APPROVED** [review]
LGTM. We will do a second iteration soon.

**Review by rashminandasahoo — APPROVED** [review]
LGTM

**Pull request merged**